### PR TITLE
TSC minutes

### PR DESCRIPTION
**Pull request by Idclip**

### CODE REVIEW

File: tsc/meetings/2023-07-25.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+Minutes from OpenVDB TSC meeting, July 25th, 2023
```
Comment (Idclip) [inline tsc/meetings/2023-07-25.md:1]: I've decided to omit the meeting count from now on because it's becoming impossible to follow exactly

Comment (jmlait) [reply tsc/meetings/2023-07-25.md:1]: If we keep the shared doc of who last wrote the meetings up to date, we also can track this number.

**Review by jmlait — APPROVED** [review]

File: tsc/meetings/2023-07-25.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Attendees: *Jeff* L., *Andre* P, *Dan* B., *Rich* J., *Nick* A.
```
Comment (kmuseth) [inline tsc/meetings/2023-07-25.md:3]: wait, I  was also attending this meeting!

**Review by kmuseth — CHANGES_REQUESTED** [review]
please add me to the list of attendees

File: tsc/meetings/2023-07-25.md (hunk 2 above)
Comment (Idclip) [reply tsc/meetings/2023-07-25.md:3]: Woops, guess I'm a bit rusty

**Pull request merged**